src/utils/script-mime-types.ts

```diff
@@ -50,5 +50,5 @@ export function getScriptParserName(typeAttrToken?: AttributeToken): BuiltInPars
 		return suffixExec[1] as unknown as 'json' | 'yaml';
 	}
 
-	return scriptTypeToParserMap.get(type);
+	return scriptTypeToParserMap[type];
```

Comment:
The reason I use maps is because this can return unexpected values. If for whatever reason `type` is equal to something on the `Object.prototype`, it won't return a string.

So if you do use the Record approach, I'd recommend using `Object.prototype.hasOwnProperty`:
```js
if (record.hasOwnProperty(type)) {
    return record[type]
}
```

Reply:
Oh my god 👀 
I didn't know that! Thanks for teaching me this. That completely changes my view of the use of Map. Until now I've always tried to avoid them as much as possible, because they actually only brought overhead. But here it actually makes sense.